rkt/status.go

```diff
@@ -55,14 +55,26 @@ func init() {
 
 	cmdStatus.Flags().Lookup("wait").NoOptDefVal = "true"
 	cmdStatus.Flags().Lookup("wait-ready").NoOptDefVal = "true"
+	cmdStatus.Flags().StringVar(&flagUUIDFile, "uuid-file", "", "read pod UUID from file instead of argument")
 }
 
 func runStatus(cmd *cobra.Command, args []string) (exit int) {
-	if len(args) != 1 {
-		cmd.Usage()
-		return 254
-	}
+	var podUUID string
 
+	if len(args) == 0 && flagUUIDFile != "" {
```

Comment:
This logic allows users to type

```
# rkt status --uuid-file=/tmp/file 01abcde
```

And then it'll print the status of 01abcde.

This should not be allowed, if `--uuid-file` is present we should not accept other arguments, like we do for other commands like `rkt stop`.